src/main/java/practice/kadai2209_7th/AirportController.java

```diff
@@ -18,115 +19,161 @@
 @RestController
 @RequestMapping("/airports")
 public class AirportController {
+
+    List<String> airportCodeList = new ArrayList<>();
+
     Map<String, List<String>> yourAirportMap = new HashMap<>();
 
-    private boolean hasAirportList = false;
+    private final String notFoundMessage = "not found";
```

Comment:
定数として扱うなら大文字スネークケースにするのがJavaの命名規則です。
```suggestion
    private final String NOT_FOUND_MESSAGE = "not found";
```

Reply:
ご指摘の箇所を修正致しました。
a9e2f69921863a807122e4a3bffe51ab370bf5d1
![image](https://user-images.githubusercontent.com/113277395/198874983-6ee863b3-07ba-42fb-acbf-7d8e1eb91940.png)